### PR TITLE
Add details about SSH with windows.

### PR DESCRIPTION
**Pull request by teesloane**

### CODE REVIEW

File: jekyll/_cci2/hello-world-windows.md [hunk 1]
```diff
@@ -164,6 +164,24 @@ Next, we run two steps: one to build the executable for Windows 10, and another
...
+It is possible to SSH into a Windows build container. This is useful for troubleshooting problems in your pipeline. Follow these steps to SSH into a Windows container:
+
+## Steps
+
+1. Ensure that you have added an SSH key to your [GitHub](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/) or [Bitbucket](https://confluence.atlassian.com/bitbucket/set-up-an-ssh-key-728138079.html) account.
+
+2. To start a job with SSH enabled, select the 'Rerun job with SSH' option from the 'Rerun Workflow' dropdown menu.
+
+3. To see the connection details, expand the 'Enable SSH' section in the job output where you will see the SSH command needed to connect:
+![SSH connection details]({{ site.baseurl }}/assets/img/docs/ssh-windows-obf.png)
```
Comment (appplemac) [inline jekyll/_cci2/hello-world-windows.md]: Will this get added in the same PR or separately?

File: jekyll/_cci2/hello-world-windows.md [hunk 2]
```diff
@@ -164,6 +164,24 @@ Next, we run two steps: one to build the executable for Windows 10, and another
...
+
+Ensure that you are passing the name of the shell you want to run when you ssh
+in. To run  `cmd.exe` in the build above you would run: `ssh -p <remote_ip> -- powershell.exe`
```
Comment (appplemac) [inline jekyll/_cci2/hello-world-windows.md]: `-- cmd.exe`.

I’d also add something like:

"The available options are:

* `-- powershell.exe`
* `-- bash.exe`
* `-- cmd.exe`

File: jekyll/_cci2/hello-world-windows.md (hunk 1 above)
Comment (teesloane) [reply jekyll/_cci2/hello-world-windows.md]: Yep, the image is included in this PR.

File: jekyll/_cci2/hello-world-windows.md (hunk 2 above)
Comment (appplemac) [reply jekyll/_cci2/hello-world-windows.md]: Sorry @teesloane — I should have been clearer — there is a typo on line 181, you are saying `cmd.exe` but are specifying `powershell.exe` in the example command.

Comment (teesloane) [reply jekyll/_cci2/hello-world-windows.md]: Woops, no my mistake! Fixing now.

**Pull request merged**